style.css

```diff
@@ -0,0 +1,231 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap");
+
+body {
+  font-family: "Poppins", sans-serif;
+  padding: 0;
+  margin: 0;
+  color: #344563;
+  font-size: 15px;
+  background-color: #e5e5e5;
+}
+
+a {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+header {
+  height: 48px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+  background-color: #fff;
+}
+
+header a {
+  font-size: 18px;
+  line-height: 20px;
+  font-weight: 700;
+  color: #6070ff;
+}
+
+a:hover {
+  color: #6070ff;
+}
+
+.menu {
+  display: flex;
+  flex-direction: column;
+  width: 24px;
+  padding: 4px;
+  cursor: pointer;
+}
+
+.menu span {
+  display: block;
+  height: 2px;
+  background-color: #6070ff;
+}
+
+.menu span:not(:last-child) {
+  margin-bottom: 5px;
+}
+
+.hero {
+  padding: 90px 24px;
+  background-image: url("./assets/header-shapes-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-color: #fff;
+}
+
+h1 {
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 700;
+  color: #172b4d;
+}
+
+.hero p,
+.info p {
+  color: #344563;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.socials {
+  display: flex;
+  padding: 0;
+  list-style-type: none;
+}
+
+.socials img {
+  margin-right: 20px;
+}
+
+.works {
+  padding: 100px 24px;
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 88px;
+}
+
+.work-card {
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  border-radius: 16px;
+}
+
+.work-card img {
+  width: 100%;
+  border-radius: 8px;
+}
+
+.details {
+  padding: 12px 0;
+}
+
+.details h2 {
+  margin: 0;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.subTitle {
+  font-size: 13px;
+  line-height: 16px;
+  padding: 0;
+  margin: 20px 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  color: #7a869a;
+}
```

Comment:
- [ ]  For better communication Instead of giving orders, make suggestions. So that the student will be fully motivated to implement the changes you requested. for example 
- Instead of saying ` you could use margin-right of the li elements instead of using justify-content: space-between to fix the spacing between the elements.` 
- just say `I think it would be better to use margin-right of the li elements instead of using justify-content: space-between to fix the spacing between the elements.`